guides/hiring/referral_policy.md

```diff
@@ -0,0 +1,62 @@
+# Employee Referral Policy
+
+The main purpose of this employee referral program policy is for our employees and all the
+People and Talent Team to use when they need detail and information about how Made Tech’s referral programs work.
+
+If you know someone who you think would be a good fit for a position at our company, feel free to refer them.
+If we end up hiring your referred candidate,
```

Comment:
This sentence appears to be missing an ending.

Reply:
yes indeed it is missing some information. 

Should say: 'If we end up hiring your referred candidate that you may receive a referral payment. Use the policy guidelines below to determine your eligibility.'